cluster-autoscaler/core/utils/utils.go

```diff
@@ -322,19 +322,6 @@ func anyPodHasHardInterPodAffinity(pods []*apiv1.Pod) bool {
 	return false
 }
 
-// ConfigurePredicateCheckerForLoop can be run to update predicateChecker configuration
-// based on current state of the cluster.
-func ConfigurePredicateCheckerForLoop(unschedulablePods []*apiv1.Pod, schedulablePods []*apiv1.Pod, predicateChecker *simulator.PredicateChecker) {
-	podsWithAffinityFound := anyPodHasHardInterPodAffinity(unschedulablePods)
```

Comment:
I'm pretty sure that's the only use of anyPodHasHardInterPodAffinity; unless I'm wrong delete that as well.

Reply:
you are right

Reply:
of course I am :D